front/js/cart.js

```diff
@@ -0,0 +1,85 @@
+//Initialisation du local storage
+let produitLocalStorage = JSON.parse(localStorage.getItem("produit"));
+console.table(produitLocalStorage);
+const positionEmptyCart = document.querySelector("#cart__items");
+
+
+// Si le panier est vide
+function getCart(){
+    if (produitLocalStorage === null || produitLocalStorage == 0) {
```

Comment:
```suggestion
    if (produitLocalStorage === null || produitLocalStorage === 0) {
```

faire des conditions strict triple égale etc voir la doc
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Strict_equality